recipes/rvlib/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "rvlib" %}
+{% set version = "0.0.1" %}
+{% set sha1 = "40110f5cf76a0fa29fa979179d66126170bf4d58" %} 
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/QuantEcon/rvlib/archive/{{ version }}.tar.gz
+  sha1: {{ sha1 }}
+
+build:
+  skip: True  # [win or not py35]
+  number: 2
```

Comment:
Please set to `0`. Everything here starts at `0`. Then we bump as needed resetting on a version bump.